lib/elasticsearch.ts

```diff
@@ -0,0 +1,41 @@
+var config  = require('../config'); 
+var request = require('request');
+
+
+// Helper function to make ES queries. Pass it a path to query and it returns
+// a promise with the decoded result
+var get_es = function(path) {
```

Comment:
`function get_es(path) {`

I suggest adding a `lint` target to the makefile and having it block a successful drone/test run (happy to help set this up, it's also documented in ES6 wiki)